lib/screens/home/home_pages_widget.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
+import 'package:survey_flutter/screens/home/home_footer_widget.dart';
+
+class HomePagesWidget extends StatelessWidget {
+  final PageController _pageController = PageController();
+
+  HomePagesWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: itemCount and background image handled in integrate
+    return PageView.builder(
+      itemCount: 4,
+      controller: _pageController,
+      itemBuilder: (BuildContext context, int index) {
+        return Stack(
```

Comment:
(Optional) Instead of using `Stack`, you can use `Container` with image decoration as a background